config/version.php

```diff
@@ -1,10 +1,10 @@
 <?php
 return array (
   'app_version' => 'v6.0.5',
-  'full_app_version' => 'v6.0.5 - build 8229-g94e723a88',
```

Comment:
These changes aren't correct - can you revert this file?

Reply:
Apologies for that, first-timer on pull requests - must have had the master version. File reverted to dev version.